Cannot find module 'ng2-bootstrap'

Apparently the reference to ng2-bootstrap is outdated. I am receiving the following error messages when trying to `ng serve` the project.

> ERROR in C:/work/meisterwerk/midge/web-app/src/app/app.module.ts (14,27): Cannot find module 'ng2-bootstrap'.
> ERROR in C:/work/meisterwerk/midge/web-app/src/app/app.component.spec.ts (7,27): Cannot find module 'ng2-bootstrap'.`

After changing the references in those two files to 'ngx-bootstrap' the project compiles without problems.

Also see https://www.npmjs.com/package/ng2-bootstrap

> this package was renamed to ngx-bootstrap

--- a/README.md
+++ b/README.md
@@ -11,7 +11,7 @@ which those tools and offers a framework for authentication right out of the box
 ## Features
 
 * An **Angular 4** app generated with angular-cli
-* Twitter Bootstrap with ng2-bootstrap
+* Twitter Bootstrap with ngx-bootstrap
 * AngularFire2 and Firebase
 * An initial implementation of an authentication service (See auth-service)
 * One component to display user information

--- a/generators/app/templates/README.md
+++ b/generators/app/templates/README.md
@@ -6,7 +6,7 @@ This project is a generator for creating new projects with the following feature
 
 * Angular 2 (based on angular-cli)
 * Firebase via AngualarFire2
-* Bootstrap with ng2-bootstrap
+* Bootstrap with ngx-bootstrap
 
 ## Configuration
 

--- a/generators/app/templates/src/app/app.component.spec.ts
+++ b/generators/app/templates/src/app/app.component.spec.ts
@@ -4,7 +4,7 @@ import {TestBed, async} from "@angular/core/testing";
 import {AppComponent} from "./app.component";
 import {AuthService} from "app/shared/auth.service";
 import {AuthServiceStub} from "app/shared/auth.service.stub";
-import {AlertModule} from "ng2-bootstrap";
+import {AlertModule} from "ngx-bootstrap";
 
 describe('AppComponent', () => {
     beforeEach(() => {

--- a/generators/app/templates/src/app/app.module.ts
+++ b/generators/app/templates/src/app/app.module.ts
@@ -11,7 +11,7 @@ import {AuthService} from "app/shared/auth.service";
 import {LoginUserComponent} from "app/login-user/login-user.component";
 import {DisplayUserComponent} from "app/display-user/display-user.component";
 import {RegisterUserComponent} from "app/register-user/register-user.component";
-import {AlertModule} from "ng2-bootstrap";
+import {AlertModule} from "ngx-bootstrap";
 import {ResetPasswordComponent} from "./reset-password/reset-password.component";
 import {Routes, RouterModule} from "@angular/router";
 import {HomePageComponent} from "./pages/home-page.component";

